test/index.test.ts

```diff
@@ -532,7 +532,6 @@ test('removes container', async (): Promise<void> => {
 
   await compose.upAll(config);
   expect(await isContainerRunning('/compose_test_nginx')).toBeTruthy();
-  expect(await isContainerRunning('/compose_test_alpine')).toBeTruthy();
```

Comment:
Regarding the issue with transient container - yeah, definitely. We could use `nginx:alpine`. This should keep the container busy :smile: 

I wasn't happy with the way we were checking against container presence as well, but was busy at the time and was not able to work on it. When migrating to TypeScript it become apparent that we could use a docker API method that directly checks a particular container without loading the whole list.
I'll look into it again in a separate PR.